packages/sit-onyx/src/components/OnyxDataGrid/types.ts

```diff
@@ -71,6 +74,11 @@ export type OnyxDataGridProps<
    * Check the Storybook examples (e.g. [Sorting](/?path=/story/data-datagrid-features--sorting)) for more details on how the features are used and configured.
    */
   features?: TFeatures;
+  /**
+   * When `async` is `true`, the data transformation of supported features is disabled and externalized.
+   * This allows for backend handling of data fetching and data transformation.
+   */
+  async?: boolean;
```

Comment:
Are you sure we want to offer a global property for this in addition to the per-feature property? Might be confusing for the users

Reply:
Lets keep this for now and wait for user feedback :D